config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -126,8 +127,7 @@ spec:
                   type: object
                 type: array
               version:
-                description: Component versions refer to specific snapshots of a component.
-                  A common scenario being the release of a component.
+                description: Version specifies the version information for the ComponentVersion.
```

Comment:
Line 142: description: ComponentVersionStatus defines the observed state of ComponentVersion

Describe that this information is added at runtime by FluxCD during the reconciliation process

Reply:
This information is added by the ocm-controller not flux.